### DISCUSSION TITLE
Extracting only first table in PDF

### DISCUSSION THREAD

**Discussion by outsu** [jsvine/pdfplumber#970] [Ask for help with specific PDFs]
![image](https://github.com/jsvine/pdfplumber/assets/70312743/cdbd37c0-96ca-4648-816a-82896e835dcf)

I want to extract only first table, where the closest one of the top of the page. But the table where bottom, sometimes it's bigger than that. So, extract_table() doesn't work. I can't use any positional setting, because PDF will be change in time, the first table may has a different numbers of rows.

So, How do I extract only first table in a page?

**Comment by jsvine** [comment]
Does this achieve your goal?:

```python
tables = page.find_tables() # Sorted from top to bottom, left to right
print(tables[0].extract())
```

**Reply by outsu** [reply]
Oh, I see the problem now. I tried to do it like that, but I was only getting an output like [['', '', '', '', '', '']]. I think it was mistaking part of the pdf for a table. It was solved when I made "tables[1]". Thank you so much!
